javascript/examples/sample.js

```diff
@@ -19,6 +19,15 @@ if (isFlagTreated(featureFlag)) {
     f2();
 }
 
+// ---------------------------------
+
```

Comment:
Can you please remove this comment line here? Other than that the code looks good!